TockLibrary.mk

```diff
@@ -29,10 +29,6 @@ $(call check_defined, LIBNAME)
 $(call check_defined, $(LIBNAME)_DIR)
 $(call check_defined, $(LIBNAME)_SRCS)
 
-ifeq ($(strip $($(LIBNAME)_SRCS)),)
-  $(error Library "$(LIBNAME)" has no SRCS?)
-endif
-
```

Comment:
?

Reply:
https://github.com/tock/libtock-c/pull/432/commits/9f7c65ba18ff242eed9ca227fb4d4233b23cd621

Reply:
Basically when running the formatter in the CI, the submodules aren't cloned. So no SRCs are found.

This avoids that error.